libkbfs/folder_branch_ops.go

```diff
@@ -3253,6 +3258,18 @@ func (fbo *folderBranchOps) applyMDUpdates(ctx context.Context,
 	return fbo.applyMDUpdatesLocked(ctx, lState, rmds)
 }
 
+func (fbo *folderBranchOps) getLatestRemoteMergedRevision() MetadataRevision {
+	fbo.muLatestRemoteMergedRevision.RLock()
+	defer fbo.muLatestRemoteMergedRevision.RUnlock()
+	return fbo.latestRemoteMergedRevision
+}
+
+func (fbo *folderBranchOps) setLatestRemoteMergedRevision(rev MetadataRevision) {
+	fbo.muLatestRemoteMergedRevision.Lock()
+	defer fbo.muLatestRemoteMergedRevision.Unlock()
+	fbo.latestRemoteMergedRevision = rev
```

Comment:
A debug log message here is probably a good idea, to say what the new merged head is.